### ISSUE TITLE
Unity games refuse to run from Steam client on some distros

### ISSUE THREAD

**Issue by TheSandBoxMKG** [ValveSoftware/Proton#2288]
System Information: [Gist](https://gist.github.com/TheSandBoxMKG/35259fffe98886e013022a6ab76981c8)
OS: Debian 9 Stretch
DE: Xfce 4.12
GPU: Intel HD Graphics 620

So... I was trying to test Yu-Gi-Oh! Duel Links when I met the same error message in found previously on [another game](https://github.com/ValveSoftware/Proton/issues/2034):
![imagen](https://user-images.githubusercontent.com/27746283/52074418-99e10c00-2568-11e9-9e31-518c785bd0da.png)
So I tested to run the game via terminal, not from the Steam client and went futher than running from the Steam client (It crashed in a black screen but following [this](https://github.com/ValveSoftware/Proton/issues/1471) I was able to get futher until the "Couldn't connect to the Steam server." message appeared).

In conclusion, I think Proton has some problems running Unity games from the Steam client in some linux configurations.

This is the log when is run from the Steam client:
[steam-601510.log](https://github.com/ValveSoftware/Proton/files/2818472/steam-601510.log)
I don't know how to make log from when the game is run directly from the terminal, but here's at leaqst the output it does in the terminal:
[Gist](https://gist.github.com/TheSandBoxMKG/6907391964f88515a96634bb201cb4d5)

**Comment by kisak-valve** [comment]
Hello @TheSandBoxMKG, Proton uses DXVK to translate D3D11 to Vulkan and you need the `mesa-vulkan-drivers` and `mesa-vulkan-drivers:i386` package to support that on Intel graphics. However, Mesa 13.0.6 is well below the [requirements](https://github.com/ValveSoftware/Proton/wiki/Requirements#directx-11-games) for Proton and not worth troubleshooting until you update to something newer. You may have some luck with adding [`PROTON_USE_WINED3D=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options.

Closing due to min-spec requirements.
